### PR TITLE
Js forms calc

### PR DESCRIPTION
**Pull request by charis-amend**

### CODE REVIEW

File: javascript-challenges/js-forms/js-forms_calculator/js/index.js [hunk 1]
```diff
@@ -19,14 +19,48 @@ function divide(a, b) {
...
+
+  const entryForA = parseInt(dataFromEntriesTurnedToObject.numberA)
+  console.log("shows new const numberForA and its entered content numberA from html:", entryForA);
+
+  const entryForB = parseInt(dataFromEntriesTurnedToObject.numberB);
+  console.log("shows entry of user for inputB", entryForB);
+
+  switch (dataFromEntriesTurnedToObject.operator) {
+    case "addition":
+      result = entryForA + entryForB;
```
Comment (mgczarnievicz) [inline javascript-challenges/js-forms/js-forms_calculator/js/index.js:43]: Don't you have a function that does this? Why not call it?

File: javascript-challenges/js-forms/js-forms_calculator/js/index.js [hunk 2]
```diff
@@ -19,14 +19,48 @@ function divide(a, b) {
...
+      break;
+
+    case "subtraction":
+      result = entryForA - entryForB;
```
Comment (mgczarnievicz) [inline javascript-challenges/js-forms/js-forms_calculator/js/index.js:47]: Don't you have a function that does this? Why not call it?

File: javascript-challenges/js-forms/js-forms_calculator/js/index.js [hunk 3]
```diff
@@ -19,14 +19,48 @@ function divide(a, b) {
...
+      break;
+
+    case "multiplication":
+      result = entryForA * entryForB;
```
Comment (mgczarnievicz) [inline javascript-challenges/js-forms/js-forms_calculator/js/index.js:51]: Don't you have a function that does this? Why not call it?

File: javascript-challenges/js-forms/js-forms_calculator/js/index.js [hunk 4]
```diff
@@ -19,14 +19,48 @@ function divide(a, b) {
...
+      break;
+
+    case "division":
+      result = entryForA / entryForB;
```
Comment (mgczarnievicz) [inline javascript-challenges/js-forms/js-forms_calculator/js/index.js:55]: Don't you have a function that does this? Why not call it?

File: javascript-challenges/js-functions/js-functions_pizza-analyzer/js/index.js [hunk 5]
```diff
@@ -1,5 +1,11 @@
+Pizza challenge
+
```
Comment (mgczarnievicz) [inline javascript-challenges/js-functions/js-functions_pizza-analyzer/js/index.js:2]: This is not finish and it shouldn't be in this PR

**Review by mgczarnievicz — CHANGES_REQUESTED** [review]

File: javascript-challenges/js-functions/js-functions_pizza-analyzer/js/index.js (hunk 5 above)
Comment (charis-amend) [reply javascript-challenges/js-functions/js-functions_pizza-analyzer/js/index.js:2]: than you Gimena I will call the function 
 about the pizza challenge -> this was a mess with my GitHub yesterday I apologise!

**Pull request merged**